pydatastructs/trees/m_ary_trees.py

```diff
@@ -0,0 +1,168 @@
+from pydatastructs.utils import M_AryTreeNode, BinomialTreeNode
+from pydatastructs.linear_data_structures.arrays import ArrayForTrees
+
+__all__ = [
+    'M_AryTree'
+]
```

Comment:
Please modify the `__init__.py` file of `trees` module to bring `M_AryTree` to main namespace. And doesn't `MAryTree` looks better, that `_` looks odd there.

Reply:
Updated.